azavea/echo-locator
# Staging Site for User Testing

## Overview

Finalized changes from Trinity and Juan Carlos.

### Demo

Optional. Screenshots, `curl` examples, etc.


### Notes

We will also be changing the env.yml file to disallow anonymous users, as we have a workaround to allow users to have accounts while testing. We will also be manually uploading some data to the assets folder (amenities icons and amenities data)

## Testing Instructions

- Click on star button to favorite a neighborhood from both the multiple neighborhood overview and the specific neighborhood details card. There should be a popup to opt in or out of text alerts for the neighborhood. Entering an invalid number will result in an error and will prompt the user to enter a valid phone number.

- In edit profile, there will be a section at the bottom detailing the users preferences for text alerts and allow them to change preferences.

- In each neighborhood view, there will be a show listings button that will cause markers to show up on the map. Clicking on each marker will render a popup detailing basic information about the listing.

- Also in each neighborhood view, there will now be an amenities bar along the top. Toggling these buttons will cause various amenities to show up.



Resolves #XXX
